src/main/java/pages/LandingPage.java

```diff
@@ -17,5 +17,16 @@ public void createAnAccountButton() {
         createAccount.click();
```

Comment:
Also as you click on the button, the new page is open -> Login page.
In that case method should return object new CutomerLoginPage(driver)  instead of being void.